relay-server/src/actors/project.rs

```diff
@@ -386,6 +386,7 @@ enum GetOrFetch<'a> {
 /// Projects can define multiple keys, in which case this structure is duplicated for each instance.
 #[derive(Debug)]
 pub struct Project {
+    aggregator: mpsc::UnboundedSender<Aggregator>,
```

Comment:
We have a circular dependency here, since aggregator already have an address of the cache project. Cache project was started first, and we could not provide the address of the aggregator here. Using channels here is one of the solution which we can use instead.